src/capture/capture.cpp

```diff
@@ -678,7 +700,16 @@ static void init_capture_dosbox_settings(Section_prop& secprop)
 	        "screenshot action will save multiple images in the specified formats.\n"
 	        "Keybindings for taking single screenshots in specific formats are also\n"
 	        "available.");
+
+	const char *video_encoders[] = {"zmbv", "ffmpeg", nullptr};
+	str_prop = secprop.Add_string("video_encoder", when_idle, "zmbv");
 	assert(str_prop);
+	str_prop->Set_values(video_encoders);
+	str_prop->Set_help(
+		"Set the video encoder to use for video capture.\n"
```

Comment:
For uniformity with other help strings, this one should produce the following output:

```
Set the video encoder to use for video capture:
  zmbv:   Lossless zlib-based compression format (default).
  ffmpeg: H.264 video and AAC audio.
```
- indentation of two spaces, not one
- default option denoted in the comment
- colon (`:`) at the end of the first line
- full stop (`.`) after each value description

Reply:
I suggest to decipher the abbreviation:
`zmbv: Zip Motion Block Video lossless compression (default).` or
`zmbv: Zip Motion Block Video DOSBox lossless compression (default).`

What is the audio compression used together with ZMBV? Also ZIP?

Can FFmpeg be used also for AV1 video and Opus audio?

Reply:
>  What is the audio compression used together with ZMBV? Also ZIP?

Audio is not compressed.  It's just raw PCM audio.

>  Can FFmpeg be used also for AV1 video and Opus audio?

I believe so, yes.  I may offer different codecs as user configurable options in the future.  I've just got it hard coded to H.264 and AAC for now.

Reply:
What @Torinde suggested is best:

```
zmbv: Zip Motion Block Video DOSBox lossless compression (default).
```

Reply:
Fully user configurable options would be nice, but in case that's difficult - I think just adding a third option of AV1+Opus would be useful in the meantime: being the most advanced "widely supported" "free" codecs.

Reply:
Fixed the help text to what @Torinde suggested.  I think those 2 codecs won't be a problem.  Really, you should be able to set video codec + audio codec + container (MP4, MKV, AVI,  etc) independently when I'm done.

Reply:
Yeah, but we should go easy on the number of options; we don't want to expose 43323 levers to the user 😅 Just what's most useful for *0%+ of people. The rest with special needs can always just write ZMBV then do manual conversion.